### PR TITLE
Feature/comet file reading

### PR DESCRIPTION
**Pull request by mwalzer**

### CODE REVIEW

File: src/openms/source/FORMAT/PepXMLFile.cpp [hunk 1]
```diff
@@ -762,6 +763,19 @@ namespace OpenMS
...
+        }
+        else if (search_engine_ == "X! Tandem")
+        {
+          peptide_hit_.setMetaValue("MS:1001330", value);
+        }
+        else if (search_engine_ == "Mascot")
+        {
+          peptide_hit_.setMetaValue("MS:1001172", value);
+        }
+        //TODO: there is no generic expect term in the CV right now
```
Comment (timosachsenberg) [inline src/openms/source/FORMAT/PepXMLFile.cpp]: why not is_a: MS:1002353 ! PSM-level e-value

File: src/openms/source/FORMAT/PepXMLFile.cpp [hunk 2]
```diff
@@ -777,12 +791,23 @@ namespace OpenMS
...
+      else if (name == "xcorr") // Sequest score
       { // and use the mvh
         value = attributeAsDouble_(attributes, "value");
-        peptide_hit_.setScore(value);
-        current_peptide_.setScoreType(name); // add "Sequest" to name?
-        current_peptide_.setHigherScoreBetter(true);
+        if (search_engine_ != "MyriMatch") //MyriMatch has also an xcorr, but we want to ignore it
+        {
+          peptide_hit_.setScore(value);
+          current_peptide_.setScoreType(name); // add "Sequest" to name?
```
Comment (timosachsenberg) [inline src/openms/source/FORMAT/PepXMLFile.cpp]: I would not use Sequest as this does not distinguish between the different Sequest subscores.

File: src/openms/source/FORMAT/PepXMLFile.cpp [hunk 3]
```diff
@@ -762,6 +763,19 @@ namespace OpenMS
         peptide_hit_.setScore(value);
         current_peptide_.setScoreType(name);
         current_peptide_.setHigherScoreBetter(false);
+        if (search_engine_ == "Comet")
+        {
+          peptide_hit_.setMetaValue("MS:1002257", value);
```
Comment (timosachsenberg) [inline src/openms/source/FORMAT/PepXMLFile.cpp]: please add shortname of CV for readability... also below

File: src/topp/IDPosteriorErrorProbability.cpp [hunk 4]
```diff
@@ -194,7 +194,25 @@ class TOPPIDPosteriorErrorProbability :
     }
     else if ((engine == "MSGFPlus") || (engine == "MS-GF+"))
     {
-      return (-1) * log10(max(hit.getScore(), smallest_e_value_));
+      if (hit.metaValueExists("MS:1002053"))
```
Comment (timosachsenberg) [inline src/topp/IDPosteriorErrorProbability.cpp]: also here ... please add short name in comment

File: src/openms/source/FORMAT/PepXMLFile.cpp (hunk 1 above)
Comment (mwalzer) [inline src/openms/source/FORMAT/PepXMLFile.cpp]: meant was not only generic but one that is parent to all. Right now, the cv is a little fragmented on the SE scores front.

**Comment by timosachsenberg** [comment]
it would be nice to have the shortnames but otherwise +1

**Comment by mwalzer** [comment]
short names coming, but I have to update and rebase first

**Comment by timosachsenberg** [comment]
+1 if pass

**Pull request merged**